sample-code/src/server.ts

```diff
@@ -48,8 +55,31 @@ app.get('/embedding', async (req, res) => {
 });
 
 app.get('/orchestration/:sampleCase', async (req, res) => {
+  const sampleCase = req.params.sampleCase;
+  const func =
+    sampleCase === 'simple'
+      ? orchestrationChatCompletion
+      : sampleCase === 'template'
+        ? orchestrationTemplating
+        : sampleCase === 'inputFiltering'
+          ? orchestrationInputFiltering
+          : sampleCase === 'outpttFiltering'
+            ? orchestrationOutputFiltering
+            : sampleCase === 'requestConfig'
+              ? orchestrationRequestConfig
+              : orchestrationChatCompletion;
+
   try {
-    res.send(await orchestrationCompletion(req.params.sampleCase));
+    const result = (await func()) as OrchestrationResponse;
+    if (sampleCase === 'inputFiltering') {
+      res.send('Input filter applied successfully');
+    } else if (sampleCase === 'outputFiltering') {
+      res.send(
+        `Output filter applied successfully with threshold results: ${result.data.module_results.output_filtering!.data!}`
+      );
+    } else {
+      res.send(result.getContent());
+    }
```

Comment:
[pp] See my other comment on the input filtering. I would like to avoid the special handling here and I would prefer, if the `getContent()` part could happen in the orchestration related code (`orchestration.ts`) just to keep it closer. In my opinion the server setup here is not really part of the sample.

Reply:
I think there could be some valuable code also here for users, namely once we support streaming and (might) want to show how to return a stream to the frontend.

But this might also be improved later once we have better error handling. Currently, input filtering gives no easy access to the filter results. Once we have that, we could add it here. Or in the "main" sample code, whatever feels more natural.